cmd/kubetoken/kubetoken.go

```diff
@@ -37,22 +43,33 @@ You can find more information on 'ExecCredential' and 'client side authenticatio
 
 	cmd.RunE = func(_ *cobra.Command, args []string) error {
 		ctx := context.Background()
-		context := args[0]
+		contextName := args[0]
 		namespace := args[1]
 
+		cacheFileName := path.Join(config.GetDotKubeFolder(), oktetoTokenCacheFileName)
```

Comment:
What if I have set my `KUBECONFIG` env var pointing to another location?

Reply:
@jLopezbarb the `KUBECONFIG` envvar points to a file, not a folder. In order to keep things correctly separated, I think it's better to have a custom file. Do you think of a way that `KUBECONFIG` could get in the way here?

This comment has made me realize that we are not handling when the `.kube` folder doesn't exist though